task_01/ex3.py

```diff
@@ -0,0 +1,4 @@
+"""For loop, built-in enumerate function, new style formatting."""
+friends = ['john', 'pat', 'gary', 'michael']
```

Comment:
Just a note - you can put blank lines to make code more readable. It looks good between some logical blocks. For example first line is dockstring, second one is "input" and others are "actions". We can bravely divide them using blank lines if we want. 
It's not a mistake and you shouldn't fix. It's just a tip.